python/cugraph/cugraph/tests/utils/test_dataset.py

```diff
@@ -128,45 +85,48 @@ def is_symmetric(dataset):
 # Tests
 
 
-# setting download_dir to None effectively re-initialized the default
-def test_env_var():
-    os.environ["RAPIDS_DATASET_ROOT_DIR"] = "custom_storage_location"
+def test_modified_env_var(tmp_path):
+    # changing env var to tmp_path's location
+    os.environ["RAPIDS_DATASET_ROOT_DIR"] = str(tmp_path)
```

Comment:
I'm wondering if it would be better to honor the `RAPIDS_DATASET_ROOT_DIR` env var a user/test env has set. You could save it so it can be restored later:
```suggestion
    orig_rdrd = os.environ.get("RAPIDS_DATASET_ROOT_DIR")
    os.environ["RAPIDS_DATASET_ROOT_DIR"] = str(tmp_path)
```